src/main.js

```diff
@@ -6,10 +6,23 @@ import { CompositeDisposable, Task } from 'atom'
 // Dependencies
 // NOTE: We are not directly requiring these in order to reduce the time it
 // takes to require this file as that causes delays in Atom loading this package
-let path
-let helpers
-let workerHelpers
-let isConfigAtHomeRoot
+//
+// The functions first assigned to the dependency variables will self-assign
+// their underlying modules once called in an animation frame.
+
+/* eslint-disable no-return-assign */
```

Comment:
See the documentation for [no-return-assign](https://eslint.org/docs/rules/no-return-assign) for the correct way to do this. Disabling a rule should be an absolute last resort.

Reply:
Oh, I think I mistook what it was yelling at me for. Haha. 

Yeah, I'll just add some parens. Allthough, I kind of think the disable above it is a nice flag that something a little funny is going on in the assignment.